file_paths.md

```diff
@@ -59,6 +59,11 @@ The files annotated by an '*' indicate that they are created by package installa
         routes.yaml                       # :route_file
         ssl                               # :ssldir                      /etc/puppet/ssl
 
+    /etc/puppetlabs/pxp-agent *
+        modules.d *                       # directory where pxp-agent modules are stored
```

Comment:
Wrong comment by the way.
Since we use different directory trees, I agree in using `modules` for both.